src/test/java/com/assignmentTest/Assignment4Test.java

```diff
@@ -0,0 +1,13 @@
+package com.assignmentTest;
+import org.junit.jupiter.api.Assertions;
+import com.assignment.Assignment4;
+import org.junit.jupiter.api.Test;
+
+public class Assignment4Test {
+    @Test
+    public void substring(){
+        System.out.println(Assignment4.getSubString("Indian Oil Corporation Ltd","Corporation"));
+        Assertions.assertEquals(Assignment4.getSubString("Indian Oil Corporation Ltd","Corporation"),"Corporation");
```

Comment:
Lets add a message in assertion, which will show up in console if the condition fails